packages/@webex/plugin-meetings/src/media/index.ts

```diff
@@ -217,10 +217,10 @@ Media.createMediaConnection = (
     {
       // TODO: RoapMediaConnection is not ready to use stream classes yet, so we pass the raw MediaStreamTrack for now SPARK-460530
       localTracks: {
-        audio: audioStream?.outputTrack,
-        video: videoStream?.outputTrack,
-        screenShareVideo: shareVideoStream?.outputTrack,
-        screenShareAudio: shareAudioStream?.outputTrack, // TODO: add type for screenShareAudio in internal-media-core SPARK-446923
+        audio: audioStream?.outputStream.getTracks()[0],
```

Comment:
we should instead change webrtc-core and have the `outputTrack` as public instead of protected

Reply:
Had a discussion with @brycetham for the same. He suggests we shall do this instead of changing the `outputTrack` to `public`